build.gradle

```diff
@@ -364,7 +364,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.65'
+  implementation 'com.github.hmcts:civil-commons:v1.0.70-CIV-13109'
```

Comment:
adding comment so we don't merge this. we need to create release tag and merge with that